swift/Samples/TicTacToe/Sources/Authentication/LoginScreen.swift

```diff
@@ -24,33 +24,23 @@ struct LoginScreen: Screen {
     var password: String
     var onPasswordChanged: (String) -> Void
     var onLoginTapped: () -> Void
-}
-
-
-extension ViewRegistry {
 
-    public mutating func registerLoginScreen() {
-        self.register(screenViewControllerType: LoginViewController.self)
+    var viewControllerDescription: ViewControllerDescription {
+        return ViewControllerDescription(
+            build: { LoginViewController() },
```

Comment:
Need to pass the screen into this init since I made the view description not always immediately call update (unless we decided that’s the way to go)